### ISSUE TITLE
Keyboard shortcuts on notifications column

### ISSUE THREAD

**Issue by sinbad** [cheeaun/phanpy#444]
I *love* Phanpy having only just discovered it. Thank you!

My only niggle is that I *love* using the keyboard shortcuts, but I can't use them on the notifications column, which has no focus. I run a multi-column setup so I can just press a number to focus a column, then j/k to navigate around, then r to reply, but I can't do that on the notifications column. I wondered if it was due to mixed content, but I can't do it even when it's set to "Only mentions".

**Describe the solution you'd like**
I'd love for the notifications column to have the same keyboard focus as all the other columns so I can quickly reply to mentions using the keyboard. It would be fine if only mentions were able to have keyboard focus, although perhaps that would cause more problems, say if the first mention was off the bottom of the screen. Ideally I'd just like to do everything on the keyboard :)

**Comment by cheeaun** [comment]
@sinbad you're right. It's due to mixed content, so I'm still figuring out what folks expect will happen when navigating around multiple content there.

As of now, you could try use the Mentions page, which is just a list of mentions, so j/k and all other shortcuts would work.

**Comment by sinbad** [comment]
@cheeaun Ah, thanks very much. I think Notifcations has so much extra utility I'll stick with it anyway, but thanks for the tip about Mentions.

Having thought about it a bit more, my 2 cents on the mixed content thing is that I think every entry should be able to have keyboard focus when navigating up/down, otherwise if you only focused mentions there would be some unintuitive jumping around (although perhaps the Shift+j/k could be re-used to mean jump only through mentions?). 

For other entries in Notifications, I think it makes sense to only allow the Enter/o keyboard shortcut to do anything.

**Comment by martin-hamilton** [comment]
In case of interest, I just did a PR to add '.' support to load new entries in Notifications - https://github.com/cheeaun/phanpy/pull/1497

BTW the j/k and o/Enter keyboard shortcuts work for me in the Notifications column in the multi-column view (if it has focus), but the 'r' shortcut doesn't. Not sure about 'x'. I might take a look...
